### ISSUE TITLE
Keyboard shortcut to toggle console

### ISSUE THREAD

**Issue by CodeThePeace** [chinchang/web-maker#376]
This is very helpful.  Perhaps some code like Ctrl+Shift+! could be added to toggle the console.

**Comment by chinchang** [comment]
I am not I understand the issue here. Are you suggesting that console should toggle with a keyboard shortcut?

**Comment by CodeThePeace** [comment]
Yes, previously, there was some issue where the console was not opening on double click as instructed by the text box that appeared with mouseover.  Now, with the toggle console button in the lower right corner the console is easily visible or hidden.  

Perhaps, a keyboard shortcut like the shortcut to clear the console would be helpful.
